src/test/java/org/folio/rest/impl/TitlesApiTest.java

```diff
@@ -121,20 +118,20 @@ void testPostTitle() {
     assertThat(postTitleRq.getId(), nullValue());
 
     Title postTitleRs = verifyPostResponse(TITLES_ENDPOINT, JsonObject.mapFrom(postTitleRq).encode(),
-      prepareHeaders(EXIST_CONFIG_X_OKAPI_TENANT_LIMIT_10, X_OKAPI_USER_ID_WITH_ACQ_UNITS, ALL_DESIRED_PERMISSIONS_HEADER), APPLICATION_JSON, HttpStatus.HTTP_CREATED.toInt()).as(Title.class);
+      prepareHeaders(EXIST_CONFIG_X_OKAPI_TENANT_LIMIT_10, X_OKAPI_USER_ID_WITH_ACQ_UNITS, ALL_DESIRED_ACQ_PERMISSIONS_HEADER), APPLICATION_JSON, HttpStatus.HTTP_CREATED.toInt()).as(Title.class);
```

Comment:
why we need changes in Title Api tests?

Reply:
It is improvement, ⁠I read book that said the Boy Scout rule "Leave the campground cleaner than you found it" :)